### PR TITLE
Update ua docs for upcoming release 2.2

### PR DESCRIPTION
**Pull request by rdwallis**

### CODE REVIEW

**Comment by christiangoudreau** [comment]
LGTM once the new release is in

**Comment by Chris-V** [comment]
LGTm

File: site/src/main/markdown/ua/nativeApps.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+There are two extra configuration steps:
+
+1. You need to provide a clientId.
+  * Since native apps don't have cookies you need to provide a unique id for the user.  You can use `device.uuid` or your own identifier.
+  
+2.  You need to turn off the CHECK_PROTOCOL task.
+
+So disable autoCreate when creating your module:
+
+```
```
Comment (Chris-V) [inline site/src/main/markdown/ua/nativeApps.md]: I added support for code highlight. Use ```java to have this code colored it'll work when my changes reach master.

**Comment by meriouma** [comment]
LGTM

**Comment by olafleur** [comment]
@rdwallis Whenever you are ready, you can merge after adding code highlighting. Thanks!

**Comment by rdwallis** [comment]
Did we release UA with this feature added yet?

**Comment by olafleur** [comment]
No, you are right, we didn't release it yet. Sorry. ;-)

@meriouma Was there a target date for the release of UA 2.2 ?

Thanks for the highlighting. ;-)

**Comment by meriouma** [comment]
I was waiting to be pinged for release, I didn't know everything was merged and ready. Is it?

**Comment by rdwallis** [comment]
Yes thinks it's ready for a release.  Sorry forgot to follow through on it.

**Comment by meriouma** [comment]
No worries, I'll plan this for this week, then I'll merge this PR when it's done

**Pull request merged**